event-log/README.md

```diff
@@ -14,7 +14,8 @@ This is a microservice which provides CRUD operations for Event Log DB.
 | GET    | ```/ping```                             | Verifies service health                                        |
 | GET    | ```/processing-status?project-id=:id``` | Finds processing status of events belonging to a project       |
 | POST   | ```/subscriptions```                    | Adds a subscription for events                                 |
-| GET    | ```/version```                          | Returns info about service version |
+| GET    | ```/version```                          | Returns info about service version                             |
+| GET    | `/events/:event-id/:project-path/payload` | Returns an event payload                                    |
```

Comment:
I added the new endpoint to the README yesterday :)

Reply:
oh oops, sorry - i didn't update my working directory 🤦🏼

Reply:
:)